src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stopwatch {
+  margin: 50px auto;
+  border: 2px solid #030;
+  border-radius: 50%;
+  width: 600px;
+  height: 600px;
+  position: relative;
```

Comment:
change css properties order according to these rules
http://codeguide.academy/html-css.html#css-order

Reply:
check all such cases